assets/src/edit-story/components/library/panes/media/local/videoOptimizationDialog.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useState } from 'react';
+import { __ } from '@web-stories-wp/i18n';
+/**
+ * Internal dependencies
+ */
+import { Plain } from '../../../../button';
+import Dialog from '../../../../dialog';
+import localStore, {
+  LOCAL_STORAGE_PREFIX,
+} from '../../../../../utils/localStore';
+import { useCurrentUser } from '../../../../../app/currentUser';
+import { useLocalMedia } from '../../../../../app/media';
+
+function VideoOptimizationDialog() {
+  const { isTranscoding } = useLocalMedia((state) => ({
+    isTranscoding: state.state.isTranscoding,
+  }));
+
+  const KEY = LOCAL_STORAGE_PREFIX.VIDEO_OPTIMIZATION_DIALOG_DISMISSED;
```

Comment:
No need to uppercase this const.

```suggestion
  const storageKey = LOCAL_STORAGE_PREFIX.VIDEO_OPTIMIZATION_DIALOG_DISMISSED;
```

Reply:
Also, could move it outside the function. Then you don't need to add it as dependency to the `useCallback`

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6443/commits/1fdf2e87c86fd1ab2367b22af4acb122853f1152